app/build.gradle

```diff
@@ -133,7 +139,9 @@ android.applicationVariants.all { variant ->
         // Our x86 builds need a higher version code to avoid installing ARM builds on an x86 device
         // with ARM compatibility mode.
 
-        if (variant.flavorName.contains("x86")) {
+        if (variant.flavorName.contains("x86_x86")) {
+            versionCode = versionCode + 3
+        } else if (variant.flavorName.contains("x86")) {
```

Comment:
Hmm, this will match `x86_64`, too. :thinking: 
I guess the `if/else` saves us here, but it would be nice if we didn't depend on ordering

Reply:
It will never hit that line - this is pretty normal. I can do something way more complicated by parsing the flavorName but I don't know if that is worth it - this logic is pretty solid.